source/Core/Drivers/OLED.cpp

```diff
@@ -405,8 +405,9 @@ void OLED::setInverseDisplay(bool inverse) {
 }
 
 // print a string to the current cursor location
-void OLED::print(const char *const str, FontStyle fontStyle) {
+void OLED::print(const char *const str, FontStyle fontStyle, uint8_t n) {
   const uint8_t *next = reinterpret_cast<const uint8_t *>(str);
+  bool cut = n ? true : false;
```

Comment:
Instead of this, could default the max length to `255`, then just expand the check on line 415 to do `while(next[0] && n--)` sort of thing

Reply:
Very creative! Haven't thought about that.